elastic/kibana
# [maps] fix sort not applied to vector tile search request

Fixes https://github.com/elastic/kibana/issues/134590

PR forwards sort to elasticsearch vector tile API request. PR also cleans up request body. Original code put fields in docvalue_fields but elasticsearch vector tile API does not support docvalue_fields so server side code cleaned up the request body. PR updates request body generation logic to use fields API so there is less work to do when creating elasticsearch vector tile API request.

To test
1) Install sample web logs data set
2) Create new map and add new "documents" layer. Select "kibana_sample_data_logs" data view, Click "Add layer"
3) Configure sort by "@timestamp" field.
4) Open inspector and verify vector tile request. includes "sort" property in the request body
